src/Transfer/Sources/NHost.php

```diff
@@ -228,6 +242,42 @@ public function convertAttribute(array $column): Attribute
         }
     }
 
+    /**
+     * Convert Index
+     * 
+     * @param string $table
+     * @return Index|false
+     */
+    public function convertIndex(array $index): Index|false
+    {
+        $pattern = "/CREATE (?<unique>UNIQUE)? INDEX (?<name>\w+) ON (?<table>\w+\.\w+) USING (?<method>\w+) \((?<columns>\w+)\)/";
```

Comment:
> `(?<unique>UNIQUE)?`

This position could also be `FULLTEXT` instead of `UNIQUE` e.g. `CREATE FULLTEXT INDEX`